tests/integration/test_backends.py

```diff
@@ -140,3 +150,13 @@ def test_statement_execution_backend_works(ws, env_or_skip):
     sql_backend = StatementExecutionBackend(ws, env_or_skip("TEST_DEFAULT_WAREHOUSE_ID"))
     rows = list(sql_backend.fetch("SELECT * FROM samples.nyctaxi.trips LIMIT 10"))
     assert len(rows) == 10
+
+
+@pytest.mark.xfail
```

Comment:
```suggestion
```
Why do you expect it to fail?

Reply:
It's actually failing during Integration testing because of some permissions issues.
See https://github.com/databrickslabs/lsql/actions/runs/8380539383